### PR TITLE
Added files via upload

### PR DESCRIPTION
**Pull request by pgmvp**

### CODE REVIEW

File: configuration.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+{
+    mDocRoot = docRoot;
+}
+
+std::string Configuration::getIpAddress()
+{
+    return mIpAddress;
+}
+
+int Configuration::getPort()
```
Comment (alexchmykhalo) [inline configuration.cpp]: Please, make getter names without "get" prefix.

File: configurable.h [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+class Configurable
+{
+public:
+    Configurable(const Configuration  &configObj);
+    Configurable(const Configurable &) = delete;
+private:
+
+    Configuration mConfigObj;
```
Comment (alexchmykhalo) [inline configurable.h]: Use reference, do not copy Configuration object.

File: configuration.cpp [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+{
+    mPort = port;
+}
+
+void Configuration::setDocRoot(std::string docRoot)
+{
+    mDocRoot = docRoot;
+}
+
+std::string Configuration::getIpAddress()
```
Comment (alexchmykhalo) [inline configuration.cpp]: All getters must be const.

**Comment by pgmvp** [comment]
Resolved

**Pull request merged**